src/components/user-profile.component.js

```diff
@@ -130,7 +137,10 @@ export const UserProfile = ({
           navigation.navigate('RepositoryList', {
             title: translate('user.repositoryList.title', language),
             user,
-            repoCount: user.public_repos > 15 ? 15 : user.public_repos,
+            repoCount:
+              user.public_repos > maxLoadingConstraints.maxPublicRepos
+                ? maxLoadingConstraints.maxPublicRepos
+                : user.public_repos,
```

Comment:
How about `Math.min`? There are 4 similar places in this file.

Reply:
Sweet!